yt/data_objects/construction_data_containers.py

```diff
@@ -2504,6 +2532,8 @@ def _export_ply(
         color_field=None,
         color_map=None,
         color_log=True,
+        color_field_max=None,
+        color_field_min=None,
```

Comment:
same here

Reply:
this one is internal (`_export_ply`).  Do you still suggest to make these optional, since it is only called from `export_ply`

Reply:
I agree that it's a less crucial for an internal function, but many projects depend on internal functionalities so I'd prefer to be over-cautious :)